### PR TITLE
Tell the body to close on exists call.

### PR DESCRIPTION
**Pull request by scottjab-stripe**
Sequins is leaking http connections, this call might be where it is happening.

r? @bartle-stripe 
cc @jerry-stripe

### CODE REVIEW

File: backend/s3_backend.go [hunk 1]
```diff
@@ -183,11 +183,11 @@ func (s *S3Backend) exists(key string) bool {
 		Bucket: aws.String(s.bucket),
 		Key:    aws.String(key),
 	}
-	_, err := s.svc.GetObject(params)
-
+	body, err := s.svc.GetObject(params)
```
Comment (bartle-stripe) [inline backend/s3_backend.go]: Nit: rename to `resp` to match line 162?

**Comment by bartle-stripe** [comment]
lgtm

**Comment by scottjab-stripe** [comment]
PTAL

**Comment by bartle-stripe** [comment]
lgtm

**Comment by scottjab-stripe** [comment]
Mac travis is being slow, tests are passing on linux, merging.

**Pull request merged**